src/Camelotia.Presentation/ViewModels/ProviderViewModel.cs

```diff
@@ -98,6 +99,18 @@ public sealed class ProviderViewModel : ReactiveObject, IProviderViewModel, IAct
                 .Log(this, $"Current path changed in {provider.Name}")
                 .ToPropertyEx(this, x => x.CurrentPath, state.CurrentPath ?? provider.InitialPath);
 
+            _getBreadCrumbs = ReactiveCommand.CreateFromTask(
+                () => provider.GetBreadCrumbs(CurrentPath)
+                );
```

Comment:
Probably worth subscribing to `_getBreadCrumbs.ThrownExceptions` as well. I guess we can either swallow all the errors that may be thrown from `provider.GetBreadCrumbs`, or toggle some default content visibility based on a boolean `ObservableAsPropertyHelper<bool>` value.

Reply:
I made it so that it falls back to the initial Path Display when getBreadCrumbs throws